src/anyio/_backends/_asyncio.py

```diff
@@ -270,13 +270,20 @@ def __exit__(
                 exc_val.exceptions if isinstance(exc_val, ExceptionGroup) else [exc_val]
             )
             if all(isinstance(exc, CancelledError) for exc in exceptions):
+
+                def uncancel() -> None:
```

Comment:
can you copy the existing pattern for version compat:

https://github.com/agronholm/anyio/blob/e2ff31573aa14df5abfb96b3c56559886310fe36/src/anyio/_backends/_asyncio.py#L86-L94
```